### DISCUSSION TITLE
Normalize by other feature subset

### DISCUSSION THREAD

**Discussion by MatanBaruch1988** [pycaret/pycaret#1239] [General]
Hi,

I have target column with SKU's quantity.
Each SKU has a different unit measure which are standing as separate feature in the data frame.
I want to trigger the normalize function via setup but it will normalize the entire quantity column without considering the unit measure feature.
Is there any way to force the normalization by measure unit (feature) instead of the entire column?

**Comment by pycaret** [comment]
@MatanBaruch1988 To normalize features based on custom measure unit as you suggested would be to process them prior to passing in the `setup` function of PyCaret.
